tutorial/advanced/t15_finetuning.ipynb

```diff
@@ -610,22 +612,11 @@
    "cell_type": "code",
    "execution_count": 15,
    "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "text/plain": [
-       "<All keys matched successfully>"
-      ]
-     },
-     "execution_count": 15,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
+   "outputs": [],
    "source": [
-    "model_torch_pretrained = lenet_torch(input_shape=torch_input_shape, classes=base_num_classes)\n",
+    "model_torch_pretrained = fe.build(model_fn=lambda: lenet_torch(input_shape=torch_input_shape, classes=base_num_classes), optimizer_fn=\"adam\")\n",
     "\n",
-    "model_torch_pretrained.load_state_dict(load(torch_pretrained_model_path))"
+    "fe.backend.load_model(model_torch_pretrained, weights_path=os.path.join(model_dir, 'lenet_torch.pt'))"
```

Comment:
we can just put the weights path into the fe.build() call to avoid having 2 lines right?

Reply:
Yes, we can directly put weights path into fe.build().

Reply:
let's show that off then